packages/snaps-execution-environments/src/node-process/index.ts

```diff
@@ -1,7 +1,9 @@
 import { executeLockdown } from '../common/lockdown/lockdown';
+import { executeLockdownEvents } from '../common/lockdown/lockdown-events';
 import { executeLockdownMore } from '../common/lockdown/lockdown-more';
 import { ChildProcessSnapExecutor } from './ChildProcessSnapExecutor';
 
+executeLockdownEvents();
```

Comment:
Since this lockdown isn't really designed for Node.js at this point lets remove it. My bad on not realizing that.

Reply:
Ok, I reverted changes that were adding this to other execution environments. It is only in the iframe.

Reply:
It should be in the offscreen bundle too, since that's run in a browser environment.

Reply:
Ok, I returned it to the `offscreen`, so it is now in the `iframe` and `offscreen`.